pages/tasks/task_details.py

```diff
@@ -14,36 +14,52 @@ class TaskDetailsPage(Base):
 
     _name_locator = (By.CSS_SELECTOR, '.content-container > h1')
 
-    _get_started_button_locator = (By.ID, 'get-started')
-    _save_for_later_button_locator = (By.ID, 'save-for-later')
     _abandon_task_button_locator = (By.ID, 'abandon-task')
     _complete_task_button_locator = (By.ID, 'complete-task')
+    _get_started_button_locator = (By.ID, 'get-started')
+    _save_for_later_button_locator = (By.ID, 'save-for-later')
```

Comment:
You know, that's exactly what I usually do for page objects too. I guess I've gotten so used to trying to put everything in alphabetical order, which I do for oneanddone code itself, I brought that tendency into this PR. I think this is fine, so I will leave it, but I wanted to explain what happened.